src/hide_if_sponsored.js

```diff
@@ -138,8 +138,12 @@ function hideIfSponsored(possibleSponsoredTextQueries, e) {
   // Look through a list of possible locations of "Sponsored" tag, and see if it matches our list of `sponsoredTexts`
   return possibleSponsoredTextQueries.some((query) => {
     const result = e.querySelectorAll(query);
+    // Checks if it is the new rule that contains `order` word
+    const isNewRule = query.indexOf("order") > -1;
```

Comment:
Can you change this to `shouldUseTextFromParent`? It won't be new rule after we land a few more commits 😄

Reply:
done!